### PR TITLE
Vs 45753 Upgrade BookingJS dependencies

### PR DESCRIPTION
**Pull request by sandip-patel-1**
Motivation
------------

UI & UX
------------

Specs
------------

Design choices
------------

Side-effects/other
------------

Tests
------------

Known issues/limitations
------------
- unable to upgrade to new webpack

Tracking
------------

Release dependencies
------------

Who should review it
------------

### CODE REVIEW

File: examples/autoload.htm [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+	<body>
+		<div id="bookingjs"></div>
+		<script
+			type="text/javascript"
+			src="http://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"
+		></script>
+		<script type="text/javascript" src="../dist/booking.js" defer></script>
+		<script type="text/javascript">
+			window.timekitBookingConfig = {
+				app_key: 'test_widget_key_YcXb8cp1mAIfbW0cYkpwPyzkaGX1Wml6',
```
Comment (bradconnolly) [inline examples/autoload.htm:23]: TODO: Change back.

File: package.json [hunk 2]
```diff
@@ -1,56 +1,82 @@
...
+	"name": "timekit-booking",
+	"description": "Make a beautiful embeddable booking widget in minutes",
+	"main": "dist/booking.min.js",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/timekit-io/booking-js.git"
+	},
+	"scripts": {
+		"watch": "webpack --watch & webpack --config webpack.config.min.js --watch && kill $!",
+		"build": "webpack && webpack --config webpack.config.min.js",
```
Comment (bradconnolly) [inline package.json:12]: Now that we're not using bail, do we need another safe guard in place?

File: package.json [hunk 3]
```diff
@@ -1,56 +1,82 @@
...
+		"node-sass": "^5.0.0",
+		"postcss": "^8.2.9",
+		"postcss-loader": "4",
+		"sass-loader": "^10.1.1",
+		"style-loader": "^2.0.0",
+		"svg-inline-loader": "0.8.2",
+		"webpack": "4",
+		"webpack-cli": "^4.6.0"
+	},
+	"resolutions": {
```
Comment (bradconnolly) [inline package.json:59]: Are these necessary?

File: examples/autoload.htm (hunk 1 above)
Comment (sandip-patel-1) [reply examples/autoload.htm]: fixed it

File: package.json (hunk 3 above)
Comment (sandip-patel-1) [reply package.json:59]: yes with new webpack version webpack-cli is needed

File: package.json (hunk 2 above)
Comment (sandip-patel-1) [reply package.json:12]: added bail option to webpack.conf.min file

**Pull request merged**